### ISSUE TITLE
Show messages when javascript blocked, Support Google AMP

### ISSUE THREAD

**Issue by LGTMXD** [zzossig/hugo-theme-zzo#152]
> Hide content when JavaScript blocked
> support google AMP
> provide Telegram instant view templates
> merge zdoc into zzo
> support markdown table merged cells [here](https://github.com/shd101wyy/markdown-preview-enhanced/issues/133)

**Comment by zzossig** [comment]
Please be more specific.
1. Hide content when JavaScript blocked 
  Can you provide me a screenshot? I don't understand.
2. support Google AMP
  I'll search more about it. I don't know what the AMP is just right now. I'll do if it worth to support it.
3. provide Telegram instant view templates
  I'm not going to make it. I have tried before. the telegram instant view just works only with plain text. But we have a code block, and markdown syntax in our post, right? I found that the instant view does not work well with that kind of content.
4. merge zdoc into zoo
  What is the `zoo`? What do you mean? I really don't understand
5. support markdown table merged cell
  I think you should ask about it here https://github.com/yuin/goldmark/. I do not control the markdown syntax. Hugo use the Goldmark to render a markdown.

So, I'm going to consider option1 and option2.

**Comment by LGTMXD** [comment]
1. 
![411757AE-8161-41DD-8E47-21A883D454EC](https://user-images.githubusercontent.com/30213081/74114878-7a52d980-4be7-11ea-9932-c4e2dd9f84f9.jpeg)



4.  

merge theme zdoc into theme zzo


hugo new —kind docs docs/xxxxx

**Comment by zzossig** [comment]
1. I still don't understand.
4. No, It's a totally different project. I'm not going to merge it. zdoc theme is made for a document and zzo theme is made for a blog. The concept and implementation are totally different behind the scenes.

**Comment by LGTMXD** [comment]
> 1. I still don't understand.
> 2. No, It's a totally different project. I'm not going to merge it. zdoc theme is made for a document and zzo theme is made for a blog. The concept and implementation are totally different behind the scenes.
> 
> You should explain more detail. So I asked you that just type the Chinese and I'll translate it to korean
> Why you need the feature? At least you can provide me some links that related the concept

https://stackoverflow.com/questions/29484142/how-to-disable-a-website-if-javascript-is-disabled

**Comment by zzossig** [comment]
Ok, I'll show messages when javascript blocked.
Regarding the amp, I have to search more about it.
I'm not going to do the rest of your requests.
There are limitations. I hope you understand.

**Comment by zzossig** [comment]
@LGTMXD 
when javascript blocked in the browser, where should I show a message?

**Comment by LGTMXD** [comment]
> @LGTMXD
> when javascript blocked in the browser, where should I show a message?


how about blew title？
![1F36EA1C-FE02-40D3-BCEB-198C1445B0C9](https://user-images.githubusercontent.com/30213081/74145947-fa565f00-4c3a-11ea-9301-7be17d5765a5.jpeg)

**Comment by zzossig** [comment]
But I don't get the point why we need to show messages when javascript blocked.
The Zzo theme works well(at least it renders well a post when javascript blocked by a browser)
Since we can render a post, no reason to show some messages .

**Comment by LGTMXD** [comment]
![123](https://user-images.githubusercontent.com/30213081/74149926-e95e1b80-4c43-11ea-9a5b-f09ba83cc1bc.gif)

**Comment by LGTMXD** [comment]
如果js被禁用，网页是无法正常显示的.

**Comment by zzossig** [comment]
Thanks for the gif.👍
I totally understand now.

So how do you think which one is better?

1. hide all contents and show some messages when javascript blocked
2. show all contents and show some messages when javascript blocked

**Comment by LGTMXD** [comment]
1

**Comment by zzossig** [comment]
@LGTMXD
Now we hide contents when javascript blocked.

And I found that the zzo theme does not work well with google amp.
The amp site checker complains to me that I have to change the javascript, style sheets.
But I don't want to change it.
So, I gave up to support google amp

![주석 2020-02-10 225624](https://user-images.githubusercontent.com/52706977/74156548-e2420800-4c59-11ea-8ea1-9a17f83e270e.png)
